Port to Python 3

pmontrasio opened the pull request:
There are two changes needed to make the code run on Python 3.
One is using `configparse` instead of `ConfigParse` and the other is using `items()` instead of `iteritems()` (it should work on Python 2 too.)
Unfortunately the new code won't run on Python 2.
Maybe it can become cross platform by following the advice at http://python-future.org/compatible_idioms.html#configparser

pmontrasio committed:
Replace ConfigParse with configparser for porting to Python3

pmontrasio committed:
Replace iteritems with items for porting to Python3

MasterFenrir commented:
I  really appreciate this, but I don't have the time to continue work on this. I mean, I've done nothing to it for months. More than a year now. And I don't think I'll ever continue on it because WHOIS servers are a bloody nightmare. But thank you for the effort!

pmontrasio commented:
And I do really appreciate your work! I agree that WHOIS is far from ideal. Probably there is no incentive for registrars to stick to a standard output format.

jhfrintrop committed:
refactor: use with statement

use with statement when working with files as this handles closing them correctly

pmontrasio committed:
Merge pull request #1 from jhfrintrop/patch-1

refactor: use with statement

pmontrasio committed:
Keep converting to Python 3, fix issue 2

Make the regex in parse.reprocess_regex work with Python 3.7+
Remove unnecessary calls to encode('utf-8')
Remove the Python 2 version of parse.is_string

pmontrasio committed:
Add email to the expected data for whirlpool.net.au

pmontrasio committed:
Make whois-oracle start with Python3

Refer to python3 in the #! of whois-oracle.py
Replace references to pwhois with whois-oracle in the README.

pmontrasio committed:
Document the current status of the module

pmontrasio committed:
Add 'registrar whois server' to the patterns to match a whois server in a response
